diagnostic.md

```diff
@@ -4,7 +4,10 @@ Using `inspect element`, write four selectors found on the main page for this
 GitHub repo.
 
 <!-- your answer starts here -->
-
+$(.h1)
+$(.ul).h2
+$(.html)
+$(.image.alt)
```

Comment:
you dont need the period before those selectors because they are not classes, just html elements